content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
+Once you have configured Google Cloud DNS, you can use browse the
+[Zones](https://console.cloud.google.com/net-services/dns/zones) page in
+your Google Cloud Platform project to setup your external domain.
+
+> **Note**
+>
+>External DNS will automatically updates DNS records if you reuse the domain
+>name, so if you delete an old cluster and create a new one it will preserve the
+>same domain configuration for the new cluster.
+
+1.  Choose a unique DNS name; you can use nested domains (for example,
+     `cluster1.acmecorp.example`). Enter the name in the `DNS Name`
+    field
+
+2.  Run the `jx create domain` command against
+    `jx create domain gke --domain cluster1.acmecorp.example`.
+
+    1.  The program prompts you to choose your Google Cloud Platform
+        project from the available list.
+
+    2.  The program prompts you to update your existing managed servers
+        to use the displayed list of Cloud DNS nameservers. Copy the
+        list for use in the next steps.
+
+3.  From the Google Cloud Platform
+    [Zones](https://console.cloud.google.com/net-services/dns/zones)
+    page, change the *Resource Record Type* to `NS`) and use the default
+    values for your domain for for *TTL* (`5`) and *TTL Unit*
+    (`minutes`).
+
+4.  Add the first nameserver to the *Name server* field
+
+5.  Click **Add item** and add any subsequent nameservers.
+
+6.  Click **Create**.
+
+7.  Configure Jenkins X for the new domain names:
+
+    -   From the directory where you extracted the Jenkins X tarball
+        archive open the `jx-requirements-gke.yml` file in a text editor
+        (such as TextEdit for macOS or gedit for Linux) and edit the
+        ingress section at the root level.
+
+            ingress:
+              domain: cluster1.acmecorp.example
+              externalDNS: true
+              namespaceSubDomain: -jx.
+              tls:
+                email: certifiable@acmecorp.example
+                enabled: true
+                production: true
+
+# Installing Jenkins X on your GKE cluster
+
+You are now ready to install Jenkins X to your cluster. From your
+command-line, change to the directory containing the pre-configured
+`jx-requirements-gke.yml` requirements file that was unarchived with
+your `jx` program binary. For example, macOS defaults to the `Downloads`
+directory:
+
+    cd /Users/acmeuser/Downloads/
+
+Run Jenkins X Boot at the command-line `jx` and include the
+pre-configured `jx-requirements-gke.yml` file:
+
+    jx boot -r jx-requirements-gke.yml
+
+The interactive command-line tool guides you through the process of
+configuring and installing the resources needed to complete your GitOps
+development environment:
+
+1.  The program prompts you to choose your Google Cloud Project from the
+    available list
+
+2.  The program prompts you for the Cluster Name, which you previously
+    chosen (in this example, `acmecluster1`).
+
+3.  The program prompts you for a `Git Owner` or your GitHub account (in
```

Comment:
It's not always GitHub. Let's not give impression that is the only Git provider.